sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/implementation/webapp/AADOAuth2AuthorizationRequestResolver.java

```diff
@@ -50,29 +50,22 @@ private OAuth2AuthorizationRequest addClaims(HttpServletRequest httpServletReque
             return oAuth2AuthorizationRequest;
         }
         // Handle conditional access policy, step 3.
-        final String conditionalAccessPolicyClaims =
-            Optional.of(httpServletRequest)
-                    .map(HttpServletRequest::getSession)
-                    .map(httpSession -> {
-                        String claims = (String) httpSession.getAttribute(Constants.CONDITIONAL_ACCESS_POLICY_CLAIMS);
-                        if (claims != null) {
-                            httpSession.removeAttribute(Constants.CONDITIONAL_ACCESS_POLICY_CLAIMS);
-                        }
-                        return claims;
-                    })
-                    .orElse(null);
+        HttpSession httpSession = httpServletRequest.getSession(false);
```

Comment:
Should we do null check here, if not session is associated with httpServletRequest and create is false.